### PR TITLE
Register UI

### PR DESCRIPTION
**Pull request by uditpd3000**
Screenshot (5)
Screenshot (6)

### CODE REVIEW

File: src/views/examples/Register.js [hunk 1]
```diff
@@ -33,126 +35,114 @@ import {
...
+    document.getElementById("submit").disabled=false;
+    document.getElementById("Passwordmatch").innerHTML= "";
+
+  }
+}
+
+function handlesubmit() {
+  //api
+  
+}
```
Comment (mridul911) [inline src/views/examples/Register.js:61]: 1. Don't define a function like this. Be it inside functional component like const functionname = () => {}.
2. Disable Register button till all fields are filled without failing any validation.
3. Kindly check the password in other ways as well like if some user make changes in password after confirm password then also it is giving same error.

File: src/views/examples/Register.js [hunk 2]
```diff
@@ -33,126 +35,114 @@ import {
...
+                    placeholder="IITK Email ID"
+                    type="email"
+                    autoComplete="off"
+                    onChange={(e) => validateEmail(e)}
+                  />
+                   <div className="text-muted text-center mt-2 mb-4">
+                       <span style={{
+                          fontWeight: 'light',
+                         color: 'red',
+                         }}>{emailError}
```
Comment (mridul911) [inline src/views/examples/Register.js:100]: Invalid is not looking good please change its position to next line or between email and password.

File: src/views/examples/Register.js (hunk 1 above)
Comment (mridul911) [inline src/views/examples/Register.js:61]: kindly gave values in console output as of now.

**Review by mridul911 — CHANGES_REQUESTED** [review]
In general once user submit form please use something similar to login actions so that we can save user using redux-session and Redirect user to dashboard.
You can look into login pull request as well how he hangle email validator and copy that here as well.

**Comment by navneet-28** [comment]
A new PR ( #11 ), with all the requested changes, has been made.

**Pull request closed without merging**